packages/quantic/scripts/build/util/sfdx.ts

```diff
@@ -27,7 +27,14 @@ export function sfdx<T = SfdxResponse>(command: string): Promise<T> {
       },
       (error, stdout) => {
         if (error) {
-          reject(error);
+          let jsonOutput;
+          try {
+            jsonOutput = JSON.parse(strip(stdout));
+          } catch (e) {
+            // The output is not JSON. The command most likely failed outside the SFDX CLI.
```

Comment:
Don't we wanna log something here? just to facilitate debugging by indicating that the command most likely failed outside the SFDX CLI?

Reply:
No, we don't need to log the error here because the promise is rejected with the error, and it is logged at a higher level.